gc/gc-iceberg-inttest/build.gradle.kts

```diff
@@ -65,12 +72,15 @@ dependencies {
 
   intTestImplementation("org.apache.spark:spark-sql_${sparkScala.scalaMajorVersion}") {
     forSpark(sparkScala.sparkVersion)
+    exclude(group = "org.apache.logging.log4j", module = "log4j-slf4j2-impl")
   }
   intTestImplementation("org.apache.spark:spark-core_${sparkScala.scalaMajorVersion}") {
     forSpark(sparkScala.sparkVersion)
+    exclude(group = "org.apache.logging.log4j", module = "log4j-slf4j2-impl")
   }
   intTestRuntimeOnly("org.apache.spark:spark-hive_${sparkScala.scalaMajorVersion}") {
     forSpark(sparkScala.sparkVersion)
+    exclude(group = "org.apache.logging.log4j", module = "log4j-slf4j2-impl")
```

Comment:
Should this go into the `forSpark`  magic?

Reply:
Maybe – I'm a freshman at the Gradle School of Wizardry & Witchcraft. Let me try your spell.